docs/development/deploying-locally/index.md

```diff
@@ -1,3 +1,50 @@
 # Deploying locally
 
 REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+
+### Instructions
+
+* (1) Clone the main REANA repository:
+```console
+$ git clone https://github.com/reanahub/reana.git
```

Comment:
- Elsewhere we are using `git` protocol in `reana-dev`' so we may use it also here.

- Beware of `origin` vs `upstream` mix up, once we switch to use `johndoe`, we'll need to amend upstream/origin in `reana` repository afterwards.

Better tell people to fork `reana` repo, then clone from there, then `pip install .` to have `reana-dev` command, then launch `reana-dev git-fork...` to have all repos personal, and only then we are ready to go with `make clone`.